### DISCUSSION TITLE
Access client from CLI

### DISCUSSION THREAD

**Discussion by Nassuel** [RaspAP/raspap-webgui#1256] [Q&A] [answered]
I was wondering if there is a way to access the AP's tabs or the client from CLI. After looking through the docs and FAQs, I was unable to find something referring to this. I could've also missed something, so please, could you point me to that? Or perhaps someone that figured this out already could provide a way to do this. 

Thank you!

**Comment by Nassuel** [comment, marked as answer]
I was able to find it. Was just looking in the wrong place 😅 It's not possible to access it from CLI. One has to connect to the wifi network and input the default credentials to make your way to the client site where you can configure things to your heart's content.

**Reply by sensemonster** [reply]
Access via ssh through the IP address of t the Raspberry Pi - not the AP address 10.etc
